db/blob/blob_file_builder.cc

```diff
@@ -372,4 +385,84 @@ void BlobFileBuilder::Abandon(const Status& s) {
   blob_count_ = 0;
   blob_bytes_ = 0;
 }
+
+Status BlobFileBuilder::PutBlobIntoCacheIfNeeded(const Slice& blob,
```

Comment:
Another question: can we have one function for inserting blob into cache? I am referring to the other function which we already have `BlobSource::PutBlobIntoCache`.

Reply:
Done